module/VuFind/src/VuFind/Sitemap/SimpleRecord.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * A minimal record class for the sitemap generator
```

Comment:
There's not really anything Sitemap-specific in here; do you think it might make sense to move this to the VuFindSearch namespace so it's potentially available for other applications as well?

Reply:
Sure thing. I just thought that maybe the class is too specialized, but VuFindSearch is probably more logical place anyway.

Reply:
It doesn't seem too specialized to me -- I'd just change the class-level comment to "A minimal record class for wrapping an array of fields" -- I think that might be useful in a variety of situations.

Reply:
Ok, I moved SimpleRecord.

Reply:
Thanks!